src/DevChatter.Bot.Core/Games/RockPaperScissors/RockPaperScissorsGame.cs

```diff
@@ -67,14 +67,25 @@ private void CleanUpAfterGame()
         public void AdjustTokens(RockPaperScissors botChoice)
         {
             List<string> winnersList = GetWinnerList(botChoice);
-            _currencyGenerator.AddCurrencyTo(winnersList, TOKENS_FOR_WINNING);
+            if (winnersList.Any())
+                _currencyGenerator.AddCurrencyTo(winnersList, TOKENS_FOR_WINNING);
         }
 
         public void AnnounceWinners(IChatClient chatClient, RockPaperScissors botChoice)
         {
             List<string> winnersList = GetWinnerList(botChoice);
-            string winners = string.Join(",", winnersList);
-            chatClient.SendMessage($"The winners are {winners}!");
+            if (winnersList.Any())
+            {
+                string winners = string.Join(",", winnersList);
+                if (winnersList.Count() > 1)
+                    chatClient.SendMessage($"The winners are {winners}!");
+                else
+                    chatClient.SendMessage($"The winners is {winners}!");
```

Comment:
The message here should read `The winner is {winners}!`

Reply:
Good catch, I'll fix it now 👍